changelog.md

```diff
@@ -1,3 +1,13 @@
+## Changes from 1.4.7 to 1.4.8
+
+### Fixed issues
+
+* [MARATHON-7697](https://jira.mesosphere.com/browse/MARATHON-7697) - CLUSTER constraint doesn't require a value.
+* [MARATHON-7724](https://jira.mesosphere.com/browse/MARATHON-7724) - Better error handling around proxy leader failure.
+* [MARATHON-7728](https://jira.mesosphere.com/browse/MARATHON-7728) - Always log exceptions when performing migration.
+* [MARATHON-7696](https://jira.mesosphere.com/browse/MARATHON-7696) - Use maxLaunchDelay to determine what delays to GC.
```

Comment:
https://github.com/mesosphere/marathon/commit/96bc8daf27822fa8c55c9c0862112e5b2a6872af
It would be better to use the commit title: `Do a better job at maintaining task failure rate limiting values per RunSpec`